kiva/basecore2d.py

```diff
@@ -353,7 +353,7 @@ def set_line_dash(self, pattern, phase=0):
             to start.  phase defaults to 0.
 
         """
-        if not alltrue(pattern):
+        if not all(pattern):
             self.state.line_state.line_dash = NO_DASH
             return
         pattern = asarray(pattern)
```

Comment:
This whole section could be re-written to be _much_ clearer in intent
```suggestion
        pattern = asarray(pattern)
        if (pattern == 0).any():
            self.state.line_state.line_dash = NO_DASH
            return
```